wilsonrljr/sysidentpy
# add all major modifications. Everything was already shown in call video with authors

- I have made changes in all funcions so we have already made a call to explain everything.
- Added new features (Extended Least Squares, Recursive Least Squares, Total Least Squares and several adaptative filters Least Mean squares based)
- New code structure
- All new documentation
- Tests in main functions
- Rewrite core functions like build information matrix and many others;
- Change Readme.md
- Add double check to every parameter
- New docstrings
- Added notebook examples
- Added more residual analysis
- Many more
